JSONSchema/definitions.json

```diff
@@ -59,6 +59,20 @@
             },
             "type": "object"
         },
+        "FillerInfo": {
```

Comment:
Do you think we should version the fillers? I think it would be a good idea to start doing that (so that we can add a test like "all the fillers have been updated to the newest version").

Reply:
you mean filler format version? 
stateTest -> GeneralStateTest 

not so many people used fillers. there has been no need to track test filler format as only me and Yoichi used it. ideally yes. test fillers could be versioned and we could have a documentation stating what are the changes.

Reply:
Yeah, I think for new formats (ie. Genesis) we should try for the "ideal" case. So I think the fillers should be versioned, and the filled should have hashes for the sources.

It will just make it easier to identify which fillers/filled need updated.

Reply:
filled tests already has hash of the filler. 
if there is going to be a vesrion of the filler format. that version would be different from genesis format version. I think we need different scheme validators here. if it is possible to split json validation into separate groups that would be great.